### DISCUSSION TITLE
Unable to change the RTP (Random Teleport) world?

### DISCUSSION THREAD

**Discussion by TheDarthDader** [Zrips/CMI#5052] [Q&A] [answered]
I'm unsure where to look to change the /rtp default world, so my users can use it and be sent to the correct world instead of an old world that is no longer in use.

**Comment by dkalaxdk** [comment, marked as answer]
If you have /rtp isn't that a custom command?
The command from CMI is /rt [player] [worldname]
Therefore a command called /rtp sounds like it might've been created with /rtp $1 [someWorldName]?

**Reply by TheDarthDader** [reply]
I guess I didn't look hard enough, you're right, it's coming from a different plugin. Thank you
